### PR TITLE
Summed up your reply and updated instructions.

### PR DESCRIPTION
**Pull request by Danrancan**
I essentially summed up your reply to me about this app in the README.file. I also walked through the installation on Ubunutu server 20.04, which did not fully work, but the new instructions should still apply to lower versions of Ubuntu. I created new commands and essentially created an installation/build tutorial for other users. Also added instructions to add the official Mono repository to Ubuntu since Ubuntu 20.04 mono repo doesn't include msbuild. In adding these additional instructions, I added commands to install the Official Mono repo using gpg --dearmor instead of apt-key, since apt-key is deprecated in Ubuntu 20.04. This will ensure that the official mono repo stays secure and up to date with modern versions of ubuntu. Glance over and approve these changes if you get a chance. I wrote this in thanks of you writing such a long winded response to my question. It is much appreciated.

### CODE REVIEW

**Comment by Strachu** [comment]
Thank you for your time but I would rather not merge this pull request.

There are just a few things I would prefer not to put into a README or write it another way, for example:
- I'm not sure suggesting putting the app in a /usr/local/bin is a correct aproach. If I'm not wrong it's intended mainly for software adhering to unix conventions such as using system-wide shared libraries just like /usr but for software you compiled yourself. This utility definitely doesn't (and will not) comply with that requirements - it has it's own dependencies deployed in the same directory. That's why I originally suggested you to put it into /opt as from I've read the /opt directory is used by convention for third party packages not adhering to unix conventions. In the end it doesn't really matter but that's just how I see it personally.
- I will not put in a README that I don't how the app will handle symlinks - I will just check it :) I also think it's enough that these issues are already addressed in a "Issues" tab on the project - they will be not be closed until I fix them.
- I would personally limit the "Building" instructions just to `nuget restore` and `msbuild` + listing of dependencies - I just assume that if someone is interested in building any application from source code he is advanced enough to know how to unzip the package or how to install mono for example (it's the project which should have those steps listed).
- I didn't know `msbuild` is that new, if that's so I will look into different approach not requiring an user to upgrade his mono version given that I've originally built OldFileRemover with an older version that the one in Ubuntu 20.04 - thanks for making me aware of it.

So to sum up - not to say your suggestion is wrong, it's just that I already have a README template I'm using for another projects and I prefer to word it my own way. I will write a better README than the current one if I do a release, that's for sure and I'll take into account some of your suggestions.

Since there seems to be an interest in this utility I'll look whether I can make a first stable release but not now. Most fixes are simple anyway, the only time consuming process is testing of all possible cases.

**Comment by Danrancan** [comment]
> Thank you for your time but I would rather not merge this pull request.
> 
> There are just a few things I would prefer not to put into a README or write it another way, for example:
> 
> * I'm not sure suggesting putting the app in a /usr/local/bin is a correct aproach. If I'm not wrong it's intended mainly for software adhering to unix conventions such as using system-wide shared libraries just like /usr but for software you compiled yourself. This utility definitely doesn't (and will not) comply with that requirements - it has it's own dependencies deployed in the same directory. That's why I originally suggested you to put it into /opt as from I've read the /opt directory is used by convention for third party packages not adhering to unix conventions. In the end it doesn't really matter but that's just how I see it personally.
> * I will not put in a README that I don't how the app will handle symlinks - I will just check it :) I also think it's enough that these issues are already addressed in a "Issues" tab on the project - they will be not be closed until I fix them.
> * I would personally limit the "Building" instructions just to `nuget restore` and `msbuild` + listing of dependencies - I just assume that if someone is interested in building any application from source code he is advanced enough to know how to unzip the package or how to install mono for example (it's the project which should have those steps listed).
> * I didn't know `msbuild` is that new, if that's so I will look into different approach not requiring an user to upgrade his mono version given that I've originally built OldFileRemover with an older version that the one in Ubuntu 20.04 - thanks for making me aware of it.
> 
> So to sum up - not to say your suggestion is wrong, it's just that I already have a README template I'm using for another projects and I prefer to word it my own way. I will write a better README than the current one if I do a release, that's for sure and I'll take into account some of your suggestions.
> 
> Since there seems to be an interest in this utility I'll look whether I can make a first stable release but not now. Most fixes are simple anyway, the only time consuming process is testing of all possible cases.

Right on. This is all valid and you are entitled to your opinion. I write tutorials on my website so maybe if you do the fixes for this I will just write a build tutorial on my site for it. Either way, thanks again. I can help out with the testing process when you get to that point. Just let me know.
